README.md

```diff
@@ -723,35 +1144,53 @@ Specifies a Manager element in the designated xml configuration.
 
 ##### `ensure`
 
-specifies whether you are trying to add or remove the Manager element. Valid values are 'true', 'false', 'present', and 'absent'. Defaults to 'present'
+specifies whether you are trying to add or remove the Manager element.
+
+Valid options: `true`, `false`, `present`, and `absent`.
+
+Default value: `present`.
 
 ##### `catalina_base`
 
-Specifies the root of the Tomcat installation. Default: `$tomcat::catalina_home`
+Specifies the root of the Tomcat installation.
+
+Default value: `$tomcat::catalina_home`
 
 ##### `manager_classname`
 
-The name of the Manager to be created. Default: `$name`
+The name of the Manager to be created.
+
+Default value: `$name`
 
 ##### `additional_attributes`
 
-Specifies any additional attributes to add to the Manager. Should be a hash of the format 'attribute' => 'value'. This parameter is optional.
+Specifies any additional attributes to add to the Manager. Should be a hash of the format 'attribute' => 'value'.
```

Comment:
I would put "Should be a hash of the format 'attribute' => 'value'." on a new line as there seems to be more than one of them, meaning translation would only need to happen once for the specific sentence.